style.css

```diff
@@ -87,3 +88,212 @@ body {
   align-items: baseline;
   justify-content: flex-start;
 }
+
+.projects-section {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  gap: 12px;
+  margin-top: 650px;
+  width: 327px;
+  height: 552px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.cards {
+  width: 295px;
+  height: 220px;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.cards-details {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  width: 295px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.title {
+  width: 295px;
+  height: 36px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.highlights {
+  display: flex;
+  flex-direction: row;
+  gap: 12px;
+  width: 236px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.highlight-text {
+  width: 55px;
+  height: 16px;
+  text-transform: uppercase;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.counter {
+  width: 8px;
+  height: 8px;
+  background: #c1c7d0;
+  flex: none;
+  flex-grow: 0;
+}
+
+.countertext {
+  width: 88px;
+  height: 16px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
```

Comment:
Great work 💯 but keep going, the main problem is the counter image, for now, it looks broken. See below 👇🏽

## Figma
![Screenshot 2023-02-17 at 21 32 57](https://user-images.githubusercontent.com/45739963/219827183-ca5bf679-e361-4226-ad0f-2e623db4bbcf.png)

## Your design
![Screenshot 2023-02-17 at 21 33 17](https://user-images.githubusercontent.com/45739963/219827198-36575754-6173-4012-bf3b-cc003c038fd3.png)